openwisp_radius/tests/test_api.py

```diff
@@ -2594,16 +2594,17 @@ def test_register_201_phone_number_empty(self):
         self.assertEqual(User.objects.count(), 1)
         url = reverse('radius:rest_register', args=[self.default_org.slug])
         phone_number = '+393664255801'
-        r = self.client.post(
-            url,
-            {
-                'username': self._test_email,
-                'email': self._test_email,
-                'password1': 'password',
-                'password2': 'password',
-                'phone_number': phone_number,
-            },
-        )
+        with self.assertNumQueries(50):
```

Comment:
:scream:

Reply:
:laughing: I was also freaked out when I saw this. Is there something we can do to reduce this?

Reply:
Also looks like Travis checks failed for this on some versions of django. This needs a different approach altogether...